src/router/employees.py

```diff
@@ -78,7 +78,7 @@ def get(self, user_id):
 
     @api.doc(body=swot_fields)
     @api.response(200, 'Status code of response')
-    def put(self, user_id):
+    def post(self, user_id):
         '''Function for handling PUT /employees/user_id requests'''
```

Comment:
Comment is incorrect.

Reply:
Yes it is. It will be fixed on the next commit.